### DISCUSSION TITLE
[Bug] php cannot use ini_set('skywalking_agent.service_name','value');

### DISCUSSION THREAD

**Discussion by cuifox** [apache/skywalking#11434] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

PHP (apache/skywalking-php)

### What happened

no

### What you expected to happen

ini_set() can used

### How to reproduce

no

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
@jmjoy Please take a look. 

@hyper-self This doesn't look like a bug, please make sure you discuss with the maintainer or have been sure what the bug is. `Can't use an API` may be a thing you expected, but not supported. This is not how to define a bug.

**Reply by cuifox** [reply]
How can I configure NGINX to host multiple websites on a single server with different service names for each website? or not use ini_set()

**Reply by wu-sheng** [reply]
I don't know PHP agent. If the readme doesn't mention, most likely it is not supported.

**Comment by jmjoy** [comment, marked as answer]
All configuration of `skywalking_agent` are system level, so you can't set them by `init_set`.

**Comment by jmjoy** [comment]
> How can I configure NGINX to host multiple websites on a single server with different service names for each website? or not use ini_set()

I suggest you to use one service name.

**Reply by cuifox** [reply]
How to distinguish?

**Reply by wu-sheng** [reply]
The answer means we don't support this case.
